build/images/base/Dockerfile

```diff
@@ -19,8 +19,8 @@ RUN set -eux; \
          *) pluginsArch=''; echo >&2; echo >&2 "unsupported architecture '$dpkgArch'"; echo >&2 ; exit 1 ;; \
     esac; \
     mkdir -p /opt/cni/bin; \
-    wget -q -O - https://github.com/containernetworking/plugins/releases/download/$CNI_BINARIES_VERSION/cni-plugins-linux-${pluginsArch}-$CNI_BINARIES_VERSION.tgz | tar xz -C /opt/cni/bin $CNI_PLUGINS
-
+    wget -q -O - https://github.com/containernetworking/plugins/releases/download/$CNI_BINARIES_VERSION/cni-plugins-linux-${pluginsArch}-$CNI_BINARIES_VERSION.tgz | tar xz -C /opt/cni/bin $CNI_PLUGINS; \
+    wget -q -O /opt/cni/bin/whereabouts https://downloads.antrea.io/whereabouts/v0.4/whereabouts-linux-${pluginsArch}
```

Comment:
@arunvelayutham sorry for yet another change but could you download `https://downloads.antrea.io/whereabouts/v0.4/whereabouts-linux-${pluginsArch}.tgz` instead and untar it? I wanted to include a copy of the Apache 2 License. Once you untar, the binary's name is simply `whereabouts`.

Reply:
@antoninbas sorry for the delay. missed to see this comment. sure, I will push it shortly

Reply:
the latest version doesn't look right, can you test it?

The url is `https://downloads.antrea.io/whereabouts/v0.4/whereabouts-linux-${pluginsArch}.tgz`: it's a tarball file that you need to untar in order to retrieve the `whereabouts` binary included in it

Reply:
yes. I didn't mean to push that change. accidently did that while rebase. I have the correct one pushed back. please chcek.